velox/type/Conversions.h

```diff
@@ -102,6 +102,10 @@ struct Converter<
       }
       if (negative) {
         for (; index < len; index++) {
+          // Ignore the fractional part for floating point/decimal.
```

Comment:
this check would allow anything after a decimal to be ignored, which means "1.abc" will be valid. Is this expected?

Reply:
Thanks! "1.abc" is invalid. Fixed in the latest commit and added test cases.